rplidar.go

```diff
@@ -113,30 +134,28 @@ func (rp *Rplidar) NextPointCloud(ctx context.Context) (pointcloud.PointCloud, e
 
 // start requests that the rplidar starts up and starts spinning.
 func (rp *Rplidar) start() {
-	rp.mu.Lock()
-	defer rp.mu.Unlock()
-
 	rp.started = true
-	rp.logger.Debug("starting motor")
-	rp.device.driver.StartMotor()
+	if rp.model != "S1" {
+		rp.logger.Debug("starting motor")
+		rp.device.driver.StartMotor()
+	}
```

Comment:
[note to self] this is specified here: https://github.com/Slamtec/rplidar_sdk#start-spinning-motor